### ISSUE TITLE
Add --AltText switch

### ISSUE THREAD

**Issue by alexsalisbury** [poshbotio/PoshBot.XKCD#1]
## Current Behavior
No alt text.

## Expected Behavior
Should be alt text if you want it.

## Possible Solution
Add --AltText switch.

## Steps to Reproduce (for bugs)
1.Run command.
2.Look for alt-text.
3.Disappointment.
4.Alcohol.

## Context
Just trying to help.

## Your Environment
* Module version used:1.0.2
* Operating System and PowerShell version:Win10
